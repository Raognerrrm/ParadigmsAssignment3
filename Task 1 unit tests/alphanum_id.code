FUNCTION aLpHa25(BeTa1ZETA)
VARS abcd, ABCD, a2c4, A2C4, a1Long2Name3Here4;
BEGIN
END
FUNCTION main()
BEGIN
END