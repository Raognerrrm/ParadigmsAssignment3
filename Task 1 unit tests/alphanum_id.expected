Program [Function "aLpHa25" (Arguments ["BeTa1ZETA"]) (Variables ["abcd", "ABCD", "a2c4", "A2C4", "a1Long2Name3Here4"]) (Block []), Function "main" (Arguments []) (Variables []) (Block [])]